frontend/src/app/main/ui/ds/utilities/swatch.cljs

```diff
@@ -0,0 +1,101 @@
+
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.main.ui.ds.utilities.swatch
+  (:require-macros
+   [app.main.style :as stl])
+  (:require
+   [app.common.data.macros :as dm]
+   [cuerdas.core :as str]
+   [rumext.v2 :as mf]))
+
+(def ^:private schema:swatch
+  [:map
+   [:background :string]
+   [:aria-label {:optional true} :string]
+   [:format {:optional true} [:enum "square" "rounded"]]
+   [:size {:optional true} [:enum "small" "medium"]]
+   [:active {:optional true} :boolean]
+   [:on-click {:optional true} fn?]])
+
+(def hsla-regex #"^hsla\((\d{1,3}),\s*(\d{1,3})%,\s*(\d{1,3})%,\s*(0|1|0?\.\d+)\)$")
+(def rgba-regex #"^rgba\((\d{1,3}),\s*(\d{1,3}),\s*(\d{1,3}),\s*(0|1|0?\.\d+)\)$")
+
+(defn- is-image? [background]
```

Comment:
Minor thing: For functions in clojure, we usually skip the `is-` prefix if we're using the `?` suffix at the end

Reply:
Fixed